packages/example/pages/_app.tsx

```diff
@@ -85,78 +86,99 @@ const RAINBOW_TERMS = 'https://rainbow.me/terms-of-use';
 const projectId =
   process.env.NEXT_PUBLIC_WALLETCONNECT_PROJECT_ID ?? 'YOUR_PROJECT_ID';
 
-const chains = [
-  mainnet,
-  polygon,
-  optimism,
-  arbitrum,
-  base,
-  zora,
-  bsc,
-  zkSync,
-  ...(process.env.NEXT_PUBLIC_ENABLE_TESTNETS === 'true'
-    ? [
-        goerli,
-        sepolia,
-        holesky,
-        polygonMumbai,
-        optimismSepolia,
-        arbitrumSepolia,
-        baseSepolia,
-        zoraSepolia,
-      ]
-    : []),
-];
-
-const { wallets } = getDefaultWallets({
-  appName: 'RainbowKit Demo',
-  projectId,
-  chains,
-});
+const walletList = getDefaultWalletList();
```

Comment:
Why'd we create a new API? `getDefaultWallets` would actually return a wallet list